packages/ia-components/components/theatres/with-youtube-spotify/audio-with-youtube-spotify.stories.js

```diff
@@ -115,8 +115,8 @@ class DataHydrator extends Component {
           <h4>Archive </h4>
           <table>
             <thead>
-              <td>Collection</td>
-              <td>Identifier</td>
+              <tr>Collection</tr>
+              <tr>Identifier</tr>
```

Comment:
I would write it with one more TR and TH.
`  <thead>
    <tr>
      <th>Month</th>
      <th>Savings</th>
    </tr>
  </thead>
`

Reply:
Was this changed? (I don't see the update.)

Reply:
not yet as this is the demo. scripts need to be updated as soon I release bookreader so demo can properly work and I will update then 👍